internal/provider/context_field_resource_test.go

```diff
@@ -0,0 +1,98 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccContextFieldResource(t *testing.T) {
```

Comment:
I've deviated from a pattern we're using in the tests where we create the config from functions but I find this much. much more readable to have it explicitly laid out like this

Reply:
I'd have to read the previous tests (probably copy pasted from an example) but this does look (ch)easy to read :cheese: